### ISSUE TITLE
Twisted SSL support

### ISSUE THREAD

**Issue by zgoda-mobica** [rollbar/pyrollbar#105]
```
:0: UserWarning: You do not have a working installation of the service_identity module: 'No module named service_identity'.  Please install it from <https://pypi.python.org/pypi/service_identity> and make sure all of its dependencies are satisfied.  Without the service_identity module and a recent enough pyOpenSSL to support it, Twisted can perform only rudimentary TLS client hostname verification.  Many valid certificate/hostname mappings may be rejected.
```

Instead of trying to [import OpenSSL](https://github.com/rollbar/pyrollbar/blob/master/rollbar/__init__.py#L110), it would be better to try to import service_identity - it already has pyOpenSSL as dependency.

**Comment by zgoda-mobica** [comment]
PR https://github.com/rollbar/pyrollbar/pull/106 fixes this issue because Treq depends on service_identity.

**Comment by ezarowny** [comment]
Addressed in #109
